### PR TITLE
refactor: usecart methods (createcart & checkout) adjustments

### PR DESCRIPTION
**Pull request by aureate-labs-team**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by jeffpdotone — CHANGES_REQUESTED** [review]
lets update version when we publish it to main

**Comment by jeffpdotone** [comment]
Please fix conflicts

File: packages/api-client/src/api/createCart/index.ts [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
+import { gql } from '@apollo/client/core'
+import { getCountry } from '../../helpers/utils'
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export default async function createCart(context, _params, _customQuery?: CustomQuery) {
-  const data = {
-    "input": {
+
+  const DEFAULT_MUTATION = gql`mutation checkoutCreate($input: CheckoutCreateInput!) {
+    checkoutCreate(input:$input){
```
Comment (jeffpdotone) [inline packages/api-client/src/api/createCart/index.ts:8]: Aren't we updating to cartCreate?

File: packages/theme/layouts/default.vue [hunk 2]
```diff
@@ -45,6 +45,7 @@ export default {
     LazyHydrate,
     TopBar,
     AppHeader,
+    LazyHydrate,
```
Comment (jeffpdotone) [inline packages/theme/layouts/default.vue]: Remove the duplicate please. :)

**Review by jeffpdotone — CHANGES_REQUESTED** [review]

File: packages/api-client/src/api/createCart/index.ts (hunk 1 above)
Comment (aureate-labs-team) [reply packages/api-client/src/api/createCart/index.ts:8]: Not now. As its impacted area is high. Right now, replaced Shopify buy deps with Appolo client

**Review by jeffpdotone — APPROVED** [review]

**Pull request merged**